extensions/src/main/java/io/lionweb/serialization/extensions/ExtraFlatBuffersSerialization.java

```diff
@@ -0,0 +1,184 @@
+package io.lionweb.serialization.extensions;
+
+import com.google.flatbuffers.FlatBufferBuilder;
+import io.lionweb.lioncore.java.language.Containment;
+import io.lionweb.lioncore.java.language.Feature;
+import io.lionweb.lioncore.java.language.Property;
+import io.lionweb.lioncore.java.language.Reference;
+import io.lionweb.lioncore.java.model.AnnotationInstance;
+import io.lionweb.lioncore.java.model.ClassifierInstance;
+import io.lionweb.lioncore.java.model.Node;
+import io.lionweb.lioncore.java.model.ReferenceValue;
+import io.lionweb.lioncore.java.serialization.FlatBuffersSerialization;
+import io.lionweb.lioncore.java.serialization.data.MetaPointer;
+import io.lionweb.serialization.flatbuffers.gen.*;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+/** It contains the logic to serialize non-standard messages. */
+public class ExtraFlatBuffersSerialization extends FlatBuffersSerialization {
+
+  public byte[] serializeBulkImport(BulkImport bulkImport) {
```

Comment:
This is a pretty long method, consider splitting it up.

Reply:
I extracted two methods from this one